roxie/roxiepipe/roxiepipe.cpp

```diff
@@ -622,7 +634,12 @@ int main(int argc, char *argv[])
 
             try
             {
-                smartSocketFactory = createSmartSocketFactory(hosts.str(), retryMode);
+#ifdef _USE_OPENSSL
+                if (useSSL)
+                    smartSocketFactory = createSecureSmartSocketFactory(hosts.str(), retryMode);
+                else
+#endif
+                    smartSocketFactory = createSmartSocketFactory(hosts.str(), retryMode);
```

Comment:
Not new, but I just noticed this module member is never released.
Should really release in a MODULE_EXIT